statistical_validation_run/run.py

```
@@ -0,0 +1,270 @@
+) -> int:
+    """The statistical validation harness entrypoint."""
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s %(levelname)s %(message)s",
+        datefmt="%H:%M:%S",
+        handlers=[logging.StreamHandler(sys.stderr)],
+    )
+
+    a = _parse_args(argv)
+
+    ts_str = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.16.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)